crates/proof-of-sql/examples/posql_db/run_example.sh

```diff
@@ -1,5 +1,5 @@
 cd crates/proof-of-sql/examples/posql_db
-cargo run --features="arrow " --example posql_db create -t sxt.table -c a,b -d BIGINT,VARCHAR
-cargo run --features="arrow " --example posql_db append -t sxt.table -f hello_world.csv
-cargo run --features="arrow " --example posql_db prove -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
-cargo run --features="arrow " --example posql_db verify -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
\ No newline at end of file
+cargo run --no-default-features --features="arrow " --example posql_db create -t sxt.table -c a,b -d BIGINT,VARCHAR
+cargo run --no-default-features --features="arrow " --example posql_db append -t sxt.table -f hello_world.csv
+cargo run --no-default-features --features="arrow " --example posql_db prove -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
+cargo run --no-default-features --features="arrow " --example posql_db verify -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
```

Comment:
Doing this would (I think) allow for both of the following to work:
```shell
./run_example.sh #run with blitzar (default)
./run_example.sh --no-default-features --features="rayon" #run without blitzar
```
```suggestion
cargo run --no-default-features --features="arrow" "$@" --example posql_db create -t sxt.table -c a,b -d BIGINT,VARCHAR
cargo run --no-default-features --features="arrow" "$@" --example posql_db append -t sxt.table -f hello_world.csv
cargo run --no-default-features --features="arrow" "$@" --example posql_db prove -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
cargo run --no-default-features --features="arrow" "$@" --example posql_db verify -q "SELECT b FROM sxt.table WHERE a = 2" -f hello.proof
```

Reply:
Well, adding "$@" allows arguments to pass through, which means you can use the same script to run it with and without blitzar,

Reply:
let me modify it and check it on and then request a PR

Reply:
To run with blitzer 
```bash
cargo run --features="arrow" "$@" -example posql_db .....
```

and by passing the arguments we can run without blitzar correct?